scripts/gui/microbe_hud.mjs

```diff
@@ -481,27 +575,194 @@ function hideWinText(){
+        "url('../../Textures/gui/bevel/compressPanel.png')";
+    panelToChange.querySelector(".expandPanel").style.backgroundImage =
+        "url('../../Textures/gui/bevel/expandPanelActive.png')";
+
+    const rows = panelToChange.querySelectorAll(".row");
+
+    for(const row of rows) {
+        const bars = row.getElementsByClassName(barToChange);
+        const title = row.getElementsByClassName(titleToChange);
+        const barValues = row.getElementsByClassName(valuesToChange);
+
+        for (const bar of bars) {
+            bar.style.display = "block";
+            let w = "";
+
+            if(panelToChange.getAttribute("id") == "compoundsPanel") {
+                w = bar.offsetWidth + 150;
+                bar.style.marginLeft = "20px";
+            } else {
+                w = bar.offsetWidth + 100;
+                bar.style.marginLeft = "20px";
+            }
+
+            bar.style.marginBottom = "4px";
+            bar.style.marginTop = "6px";
+            bar.style.width = w + "px";
+        }
+
+        for (const tit of title) {
+            tit.style.visibility = "visible";
+        }
+
+        for (const barValue of barValues) {
+            if(panelToChange.getAttribute("id") == "compoundsPanel") {
+                barValue.style.left = "120px";
+            } else {
+                barValue.style.left = "100px";
+            }
+        }
+    }
+}
+
 //! Updates the GUI bars
 //! values needs to be an object with properties set with values for everything
 function updateMicrobeHUDBars(values){
+    let progress = 0;
+
+    // The bars
     document.getElementById("microbeHUDPlayerHitpoints").textContent =
         values.hitpoints;
     document.getElementById("microbeHUDPlayerMaxHitpoints").textContent =
         values.maxHitpoints;
-    document.getElementById("microbeHUDPlayerHitpointsBar").style.width =
-        common.barHelper(values.hitpoints, values.maxHitpoints);
-
-    // TODO: remove this debug code
-    document.getElementById("microbeHUDPlayerATP").textContent =
-        values.compoundATP.toFixed(1);
 
-    // The bars
     document.getElementById("microbeHUDPlayerATPCompound").textContent =
          values.compoundATP.toFixed(1);
     document.getElementById("microbeHUDPlayerATPMax").textContent =
          values.ATPMax;
-    document.getElementById("microbeHUDPlayerATPBar").style.width =
-         common.barHelper(values.compoundATP, values.ATPMax);
+
+    const valueAtp = common.barHelper(values.compoundATP, values.ATPMax).replace("%", "");
+    const valueHp = common.barHelper(values.hitpoints, values.maxHitpoints).replace("%", "");
+
+    const circles = document.querySelectorAll("#circleBars");
+
+    // ! instead of using totalProgress var, two hardCoded value are used
+    // They are in thrive_gui.html at line 117 and 134.
+    // two loops could be used but this need draw two differents svg for each circle
+
+    for(const circle of circles) {
+
+        progress = 100 - valueAtp;
```

Comment:
`progress` is first used here. The variable should be defined here as `let progress = 100 - valueAtp;`